src/libra_py/dynamics/qtag/QTAG_pots.py

```diff
@@ -0,0 +1,323 @@
+"""
+..module:: QTAG_pots
+  :platform: Unix, Windows
+  :synopsis: This module contains functions for computing potential values for various models.
+
+..moduleauthors :: Matthew Dutra
+"""
+
+import sys
+from liblibra_core import *
+
+#External module for NaFH system, not needed for general QTAG.
```

Comment:
this looks like a "model" Hamiltonian - can you add it to the libra_py/models, also using the format and conventions used there? as you say, this is not specific to QTAG, but could be studied with other methods